c/ntdrivers/diskperf_true-unreach-call.i.cil.c

```diff
@@ -4,6 +4,21 @@ extern char __VERIFIER_nondet_char(void);
 extern int __VERIFIER_nondet_int(void);
 extern long __VERIFIER_nondet_long(void);
 extern void *__VERIFIER_nondet_pointer(void);
+
+
+extern void abort (void) __attribute__ ((__nothrow__ , __leaf__)) __attribute__ ((__noreturn__));
+/*
+ * This function is used to detect and prevent undefined behaviour (by aborting the execution)
+ * caused by overlaping memory blocks passed to the memcpy function.
+ */
+void memcpy_guard__abort_when_overlaping_memory_blocks(void* p1, void* p2, unsigned int n)
+{
+    if((char*)p1 + n <= (char*)p2 || (char*)p2 + n <= (char*)p1)
+        return;
+    abort();
```

Comment:
I just noticed that abort() function is not described in SV-COMP rules, so I think it is better to use __VERIFIER_assume((char*)p1 + n <= (char*)p2 || (char*)p2 + n <= (char*)p1)